_includes/general/nav.html

```diff
@@ -1,24 +1,32 @@
 <aside id="gnav" class="fx-col-100-xs fx-col-25-m">
   {% if page.hidelang %}{% else %}{% include langselect.html %}{% endif %}
+
+<!--
+Default lang is: {{ site.default_lang }}
+Current lang is: {{ page.lang }}
+-->
+
+  {% if page.lng == site.default_lang %} {% comment %} Not sure why this isn't true for en. {% endcomment %} 
```

Comment:
> `Current lang is: {{ page.lang }}`

If so, should this be
```suggestion
  {% if page.lang == site.default_lang %} {% comment %} Not sure why this isn't true for en. {% endcomment %} 
```
?